wsmaster/che-core-api-factory-github/src/main/java/org/eclipse/che/api/factory/server/github/GithubAuthorizingFileContentProvider.java

```diff
@@ -37,4 +38,19 @@
   protected String formatAuthorization(String token) {
     return "token " + token;
   }
+
+  @Override
+  protected boolean isPublicRepository(GithubUrl remoteFactoryUrl) {
+    try {
```

Comment:
it works but this method should be added to GithubApiClient directly no ?
https://github.com/eclipse-che/che-server/blob/21edcc0cdac7add1b6e705c8fe0118b2e150d0da/wsmaster/che-core-api-factory-github/src/main/java/org/eclipse/che/api/factory/server/github/GithubApiClient.java

Reply:
Now it's just a small helper, which if moved to ApiClient should be expanded to have DTO for resonse, support authentication etc. so i stick to simpler variant.